shipyard/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Shipyard logs to Datadog",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Shipyard logo",
+        "image_url": "images/shipyard.svg"
+      }
```

Comment:
No need to add the logo here, it will automatically appear on your tile. Instead, can you add the sample dashboard image here? Feel free to add other images of your platform as well if you think it's valuable for customers.